Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -706,6 +782,35 @@ theorem mem_prod {x y z : ZFSet.{u}} : z ∈ prod x y ↔ ∃ a ∈ x, ∃ b ∈
 theorem pair_mem_prod {x y a b : ZFSet.{u}} : pair a b ∈ prod x y ↔ a ∈ x ∧ b ∈ y := by
   simp
 
+@[simp]
+theorem prod_empty_right {x : ZFSet} : x.prod ∅ = ∅ := by
+  ext z; simp
+
+@[simp]
+theorem prod_empty_left {x : ZFSet} : ZFSet.prod ∅ x = ∅ := by
+  ext z; simp
+
+theorem prod_subset_prod_iff {A C B D: ZFSet} (hA : A ≠ ∅) (hC : C ≠ ∅) :
```

Comment:
I can't find [this one for regular `Set`s](https://loogle.lean-lang.org/?q=%28_+%C3%97%CB%A2+_%29+%E2%8A%86+%28_%C3%97%CB%A2_%29); could you add it for `Set` too?